pages.pt_BR/common/pactree.md

```diff
@@ -0,0 +1,24 @@
+# img2pdf
+
+> Ferramenta de conversão sem perdas de imagens para PDF.
+> Mais informações: <https://gitlab.mister-muffin.de/josch/img2pdf>.
+
+- Converter múltiplas imagens para um único PDF, cada imagem sendo uma página:
+
+`img2pdf {{caminho/da/imagem1}} {{caminho/da/imagem2.jpg}} --output {{caminho/do/arquivo}}`
+
+- Converter para PDF apenas o primeiro quadro de uma imagem com múltiplos quadros:
+
+`img2pdf {{caminho/do/arquivo.gif}} --first-frame-only --output {{path/to/file.pdf}}`
+
+- Auto-orientar a imagem, usar uma página A4 em modo paisagem e uma borda de 2cm horizontalmente e 5.1cm verticalmente:
+
+`img2pdf {{caminho/do/arquivo.jpg}} --auto-orient --pagesize {{A4^T}} --border {{2cm}}:{{5.1cm}} --output {{caminho/do/arquivo.pdf}}`
+
+- Encolher apenas imagens maiores para um retângulo de 10cm por 15cm dentro de uma página de 30x20cm:
+
+`img2pdf {{caminho/do/arquivo.tiff}} --pagesize {{30cm}}x{{20cm}} --imgsize {{10cm}}x{{15cm}} --fit {{shrink}} --output {{caminho/do/arquivo.pdf}}`
+
+- Converter uma imagem para PDF e especificar os metadados do arquivo resultante:
+
+`img2pdf {{caminho/do/arquivo.png}} --title {{título}} --author {{autor}} --creationdate {{1970-01-31}} --keywords {{palavra-chave1 palavra-chave2}} --subject {{assunto}} --output {{caminho/do/arquivo.pdf}}`
```

Comment:
```suggestion
`img2pdf {{caminho/do/arquivo.png}} --title {{título}} --author {{autor}} --creationdate {{1970-01-31}} --keywords "{{palavra-chave1 palavra-chave2}}" --subject {{assunto}} --output {{caminho/do/arquivo.pdf}}`
```

Reply:
Why the quotation marks? The original version didn't use them (should I change it?). I could use _ instead of -.

Reply:
:thinking: How are the keyword treated when are space separated? are there 2 diff keywords?

Reply:
If it works this way, you don't need to change it. But it really confuses me, _how_ it works.

Reply:
From the source code:
```Python
metaargs.add_argument(
        "--keywords",
        metavar="kw",
        type=str,
        nargs="+",
        help="Sets the keywords metadata value (can be given multiple times)",
)
```
This looks like the example above shouldn't even work and it should be `--keyword {{keyword1}} --keyword {{keyword2}}`.

Reply:
I tested this and it seems like `--keyword {{keyword1 keyword2}}` saves the two words as different keywords. `--keyword "{{keyword1 keyword2}}"` will save them to just one keyword according to Okular but two according to `exiftool`. `--keyword {{keyword1}} --keyword {{keyword2}}` doesn't work, only keyword2 will be registered.

Reply:
That is horrible, crappy, confusing and doesn't make any sense at all. But your page is correct, marking this as resolved.

Reply:
```suggestion
`img2pdf {{caminho/do/arquivo.png}} --title {{título}} --author {{autor}} --creationdate {{1970-01-31}} --keywords {{palavra_chave1 palavra_chave2}} --subject {{assunto}} --output {{caminho/do/arquivo.pdf}}`
```